profiles/arch/alpha/package.use.mask

```diff
@@ -11,6 +11,7 @@ sci-libs/umfpack cholmod
 media-libs/openal sndio
 media-video/ffmpeg sndio
 media-sound/moc sndio
+media-video/mplayer sndio
```

Comment:
You don't need to add mplayer in this case, mplayer-9999 doesn't have ~alpha in it's keywords.